lib/network/default.nix

```
@@ -0,0 +1,107 @@
+{ lib }:
+{
+  ipv6 = {
+    /**
+      Normalize an IPv6 address by removing the "::" compressions and padding
+      them with the necessary number of zeros to create the correct address. Converts
+      an address from the string to the list of strings which then can be parsed using
+      `_parseNormalized`. Throws an error when the address is malformed.
+
+      # Type: String -> [ String ]
+    */
+    _normalizeIpv6 =
```

```suggestion
    _normalizeIp =
```
This is already defined in the ipv6 scope, so I think it's preferable to have:
```nix
lib.network.ipv6._normalizeIp
```
over
```nix
lib.network.ipv6._normalizeIpv6
```